linux_os/guide/system/auditing/auditd_configure_rules/audit_unsuccessful_file_modification/audit_rules_unsuccessful_file_modification_detailed_open/rule.yml

```diff
@@ -0,0 +1,62 @@
+documentation_complete: true
+
+prodtype: rhel7,fedora
+
+title: 'Record Detailed Unauthorized Access Attempts to Files (unsuccessful) - open'
+
+description: |-
+    The audit system should collect detailed unauthorized file
+    accesses for all users and root.
+    If the <tt>auditd</tt> daemon is configured to use the <tt>augenrules</tt>
+    program to read audit rules during daemon startup (the default), add the
+    rules below to a file with suffix <tt>.rules</tt> in the directory
+    <tt>/etc/audit/rules.d</tt>.
+    If the <tt>auditd</tt> daemon is configured to use the <tt>auditctl</tt>
+    utility to read audit rules during daemon startup, add the rules below to
+    <tt>/etc/audit/audit.rules</tt> file.
+    <pre>
+    -a always,exit -F arch=b32 -S open -F a2&amp;0100 -F exit=-EACCES -F auid>=1000 -F auid!=unset -F key=unsuccesful-create
+    -a always,exit -F arch=b32 -S open -F a2&amp;0100 -F exit=-EPERM -F auid>=1000 -F auid!=unset -F key=unsuccesful-create
+    -a always,exit -F arch=b32 -S open -F a2&amp;01003 -F exit=-EACCES -F auid>=1000 -F auid!=unset -F key=unsuccesful-modification
+    -a always,exit -F arch=b32 -S open -F a2&amp;01003 -F exit=-EPERM -F auid>=1000 -F auid!=unset -F key=unsuccesful-modification
+    -a always,exit -F arch=b32 -S open -F exit=-EACCES -F auid>=1000 -F auid!=unset -F key=unsuccesful-access
+    -a always,exit -F arch=b32 -S open -F exit=-EPERM -F auid>=1000 -F auid!=unset -F key=unsuccesful-access
+    </pre>
```

Comment:
hmmmm...... the amount of rules here tells me that we need to break this apart into separate rules so that the CCE can be correctly assigned.

Reply:
This could be broken in rules to verify existence of the correct rules, and one rule to verify order of them.
But the rule verifying the order would have as much objects as this one, just with less tests.

Reply:
Well, how do you suggest to split the Rules?
I suggest by filter, those a2&0100` stuff.

Reply:
Looking at it, I think that makes sense.